packages/@aws-cdk/aws-rds/lib/instance-engine.ts

```diff
@@ -507,6 +507,8 @@ class PostgresInstanceEngine extends InstanceEngineBase {
  * (those returned by {@link DatabaseInstanceEngine.oracleSe}
  * and {@link DatabaseInstanceEngine.oracleSe1}).
  * Note: RDS will stop allowing creating new databases with this version in August 2020.
+ *
+ * @deprecated instances can no longer be created with these engine versions. See https://forums.aws.amazon.com/ann.jspa?annID=7341
```

Comment:
Yeeeaaahhhh... we have a tough decision to make here.

I think in the discussion room we said that we shouldn't mark these as `@deprecated`, because that means they will get removed in v2 and that means existing applications (with databases that were deployed previously and continue to run fine) will be broken with no way to recover.

So I guess just a note by itself will have to do.

I'm also fine to invent a new annotation like `@unsupported` or something. It will render as regular text everywhere, until we add support for it to the docs to add a symbol or a color or something to distinguish it from the rest.

Reply:
IMHO, simply add 'DEPRECATED' or 'NOT RECOMMENDED FOR USE' in all caps.

Reply:
Does this change the conversation at all?

> Starting on November 1, 2020, Amazon RDS Oracle will automatically upgrade DB instances from 11.2.0.4 on SE1 LI to the latest Release Update (RU) on RDS for Oracle Database 19c SE2 LI during the scheduled maintenance window.

I don't think any reasonable plan is for v2 to be released before Nov 1. For existing applications past that point, keeping the SE1 references would actually mean a `cdk diff` would show a downgrade, and attempting to deploy that downgrade would fail, so existing applications are going to need to update regardless.

Reply:
I'm ok going with `@deprecated` if you would like this to be removed for CDKv2. Totally!

Reply:
@nija-at  - That's my preference. I'm going to let @skinny85 -- as the primary RDS owner at the moment, and someone who had strong opinions on `@deprecated` and v2 -- the chance to chime in as well.

Reply:
> Does this change the conversation at all?
> 
> > Starting on November 1, 2020, Amazon RDS Oracle will automatically upgrade DB instances from 11.2.0.4 on SE1 LI to the latest Release Update (RU) on RDS for Oracle Database 19c SE2 LI during the scheduled maintenance window.
> 
> I don't think any reasonable plan is for v2 to be released before Nov 1. For existing applications past that point, keeping the SE1 references would actually mean a `cdk diff` would show a downgrade, and attempting to deploy that downgrade would fail, so existing applications are going to need to update regardless.

That does change things, yes. Since RDS will be changing these in the backend themselves, I'm fine with removing these in v2.0, and thus marking them as deprecated here.